pytorch_lightning/strategies/launchers/spawn.py

```diff
@@ -48,6 +48,10 @@ class _SpawnLauncher(_Launcher):
     def __init__(self, strategy: Strategy) -> None:
         self._strategy = strategy
 
+    @property
+    def is_interactive_compatible(self) -> bool:
+        return False  # TODO: the return value should depend on 1) start_method 2) CUDA vs. CPU
```

Comment:
can do this once #11966 refactor is done. 

Here, it should be `return self.start_method == "spawn" or strategy.root_device == cuda`
See #7550 for more context.

Reply:
AFAIK, it should work for xla spawn as well

Reply:
yeah definitely, for xla spawn we return True.

Reply:
for the xla spawn launcher, we can just return True